alibi_detect/od/processor/base.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+
+
+class BaseProcessor():
+    def __init__(self, detectors):
+        self.detectors = detectors
+
+    def __call__(self, X):
+        B, _ = X.shape
+        results = np.empty((B, len(self.detectors)))
+        for ind, detector in enumerate(self.detectors):
+            results[:, ind] = detector.score(X)
```

Comment:
Does this break when you have an ensemble of ensembles (e.g. an ensemble incl. a KNN detector with `k=[1,2,3,4,5]` and some other detector)?